libs/tenant_size_model/src/svg.rs

```diff
@@ -6,13 +6,13 @@ const SVG_WIDTH: f32 = 500.0;
 struct SvgDraw<'a> {
     storage: &'a StorageModel,
     branches: &'a [String],
-    seg_to_branch: &'a [usize],
+    seg_to_branch: &'a [(usize, bool)],
```

Comment:
Didn't find a good way to propagate the info regarding whether a point is a lease.